docs/src/howto-guides/025_config_management_plugin.md

```diff
@@ -0,0 +1,92 @@
+# How to manage configuration files with Cumulocity
+
+With `thin-edge.io`, you can manage config files on a device by using the [Cumulocity configuration management feature](https://cumulocity.com/guides/users-guide/device-management/#managing-configurations) as a part of Device Management.
+
+If you are new to the Cumulocity **Configuration Management** feature,
+we recommend you to read [the Cumulocity user guide](https://cumulocity.com/guides/users-guide/device-management/#managing-configurations) along with this how-to guide.
+
+## Installation of `c8y_configuration_plugin`
+
+To enable the feature, first you need to install the `c8y_configuration_plugin` binary on your device.
+
+### With debian package
+
+For Debian distribution OS, we provide the `c8y_configuration_plugin_<version>_<arch>.deb` package as a release asset [here](https://github.com/thin-edge/thin-edge.io/releases).
+
+If you have used `get_thin_edge_io.sh` script to install `thin-edge.io`, this package is installed, by default.
+You can skip this section.
```

Comment:
Instead of having this *inside* the Debian section, I would add a 3rd sections. And even a  4th to separate the from source section.

* Install script (it requires `apt`)
* Using the Debian package
* For non-Debian OS distribution
* From source

Reply:
I would also add the command lines for each case.